cmd/gen-manifests/main.go

```diff
@@ -305,9 +317,21 @@ func mergeOverrides(base, overrides composeRequest) composeRequest {
 }
 
 func main() {
+	// common args
 	outputDirFlag := flag.String("output", "test/data/manifests.plain/", "manifest store directory")
 	nWorkersFlag := flag.Int("workers", 16, "number of workers to run concurrently")
 	cacheRootFlag := flag.String("cache", "/tmp/rpmmd", "rpm metadata cache directory")
+
+	// manifest selection args
+	archesFlag := new(multiValue)
+	flag.Var(archesFlag, "arches", "architectures")
```

Comment:
Please, add a comment about commas.

Also, if you change `multiValue` a bit, you can drop the comma thing and just allow the user to use `-arches` multiple times. https://github.com/osbuild/osbuild-composer/blob/main/cmd/osbuild-upload-azure/main.go#L52 does something similar.

Reply:
I like having the option to put a bunch on a list without needing to type out the argument multiple times, but allowing multiples is also useful at times.  Would it be weird if it supported both?  In other words, if
```
... -arches x86_64,aarch64 -arches s390x
```
was a valid way to specify 3 architectures?